robot/basestation/templates/elements/controls/PdsControls.html

```diff
@@ -118,26 +118,30 @@
       <tr>
         <td>
           <button title="Reset general flags" id="reset-general-flags-button" type="button" class="btn btn-primary btn-md">
-            <strong>Reset General Flags</strong>
+            <strong class"pds-text">Reset General Flags</strong>
```

Comment:
missed a couple of `=`s here and on line 126 for "Reset Current Flags"

Reply:
Fixed